releases/latest/kernel/helpers/runtime/docker-server.sh

```diff
@@ -1,65 +1,4 @@
-#!/bin/bash
```

Comment:
the `importKeyCert` functionality was probably between when you cloned the repo and when the PR was submitted.  Can you please merge it first?